chart/templates/ingress.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.hostnameOverride -}}
```

Comment:
This should not be optional, it is always required - Use domain instead of hostnameOverride, overrides can be confused with helm defaults